seeds/1_users.js

```diff
@@ -1,13 +1,20 @@
 'use strict';
 
-var dropTables = require('../migrations/1_create_users_projects_files').down;
-var createTables = require('../migrations/1_create_users_projects_files').up;
+// FIXME: TODO: We may want to make these user accounts less
+//              "real people" and more "generic developer"
 
 exports.seed = function(knex, Promise) {
-  return dropTables(knex)
-    .then(function() { return createTables(knex); })
+  var users = [
+    { name: 'ag_dubs' },
+    { name: 'k88hudson' },
+    { name: 'jbuckca' },
+    { name: 'pomax' },
+  ];
 
-    .then(function() { return knex('users').insert({ name: 'ag-dubs' }); })
-    .then(function() { return knex('users').insert({ name: 'k88hudson' }); })
-    .then(function() { return knex('users').insert({ name: 'jbuckca' }); });
+  return Promise.join(
+    knex.insert(users[0]).into('users').catch(function() { return knex.table('users'); }),
+    knex.insert(users[1]).into('users').catch(function() { return knex.table('users'); }),
+    knex.insert(users[2]).into('users').catch(function() { return knex.table('users'); }),
+    knex.insert(users[3]).into('users').catch(function() { return knex.table('users'); })
+  );
```

Comment:
catches are for ignoring duplication errors if `npm run knex` or `npm run test` is run after the database was already built.